content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -2,36 +2,32 @@
 
 You can add models from any ML project to the model registry. To add a model to
 your model registry, Iterative Studio creates an annotation for it in an
-`artifacts.yaml` file in your Git repository. If you are using the GTO command
-line tool, you can also add models from the [CLI][gto]. To add models using
-Iterative Studio, watch this tutorial video or read on below:
+`artifacts.yaml` file in your Git repository. If you are using the [GTO] command
+line tool, you can also add models [from the CLI]. To add models using Iterative
+Studio, watch this tutorial video or read on below:
 
 https://www.youtube.com/watch?v=szzv4ZXmYAs
 
-1. Click on `Add a model`
-2. Select the project to which you want to add the model. If your model file or
-   the DVC file for your model exists in a Git repository, select the project
-   that connects to this Git repository. If your model file resides in a remote
-   storage (eg, S3, GCS, etc.), select the project whose Git repository you want
-   to add the model to.
+1. Click on `Add a model`.
+
+2. Select a [connected repository] to which you want to add the model.
 
    <admon>
 
-   Only repositories that you have connected to Iterative Studio are available
-   in the `Add a model` form. To connect your desired repository to Iterative
-   Studio, go to the `Projects` tab and
-   [create a project that connects to this Git repository](/doc/studio/user-guide/projects-and-experiments/create-a-project).
-   Then you can come back to the model registry and add the model.
+   If your model file or the `.dvc` file for your model exists in a Git repo,
+   select the project that connects to it. If your model file resides in remote
+   storage (S3, GCS, etc.), select the project to whose Git repo you want to add
+   the model.
 
    </admon>
 
 3. Enter the path of the model file as follows:
 
    - If the model file is in the Git repository, enter the relative path of the
-     model from the repository root.
-   - If the model file is in a remote storage but is DVC-tracked, enter the path
-     of the DVC file for your model.
-   - If the model file is in a remote storage and is not DVC-tracked, enter the
+     model (from the repository root).
+   - If the model file is in remote storage but is DVC-tracked, enter the
+     project path of the corresponding `.dvc` file.
+   - If the model file is in remote storage and is not DVC-tracked, enter the
      absolute path of the model file.
    - If you use [MLEM] to save your model, use the path to the binary file that
      MLEM generates. After you have run
```

Comment:
These may also need updating per https://github.com/iterative/dvc.org/pull/4065#pullrequestreview-1155663567 .

Reply:
Yes, this will change as per LWMM with data.

Reply:
Sounds good. Fell free to continue this PR or merge it and work elsewhere as you prefer and can. Lmk if I can help with anything else. Thanks